.ci/packaging.groovy

```diff
@@ -203,6 +203,7 @@ pipeline {
                   'metricbeat',
                   'packetbeat',
                   'x-pack/auditbeat',
+                  'x-pack/dockerlogbeat',
```

Comment:
Do we want to add it to the packaging? Technically there is no "packaging" for this, it's just built and uploaded to docker hub.

Reply:
It's part of AMD64 packaging jobs as well. It is nice to have this job to make sure we are not breaking anything.